.gitignore

```diff
@@ -31,8 +33,3 @@ phpunit.phar
 
 # vagrant runtime
 /.vagrant
-
-# codeception generated files
```

Comment:
I did not, the separate .gitignore files were already there. But nevertheless it makes a lot of sense (see #202 for my argumentation)